app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryFragment.kt

```diff
@@ -142,6 +144,18 @@ class LibraryFragment : BaseFragment() {
     }
   }
 
+  private fun noInternetSnackbar() {
+    view?.snack(
+      R.string.no_network_connection,
+      R.string.settings,
+      { (::openNetworkSettings)(Intent(Settings.ACTION_WIFI_SETTINGS)) }
+    )
+  }
+
+  private fun openNetworkSettings(intent: Intent) {
+    startActivity(intent)
```

Comment:
This should just be `startActivity(Intent(Settings.ACTION_WIFI_SETTINGS))`

Reply:
I'll update it. Sorry, I misinterpreted the previous comment.

Reply:
@macgills updated.